staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -62,6 +62,22 @@ var (
 		},
 		[]string{"resource"},
 	)
+	watchCacheCapacityIncreaseTotal = metrics.NewCounterVec(
```

Comment:
Sorry - apparently I wasn't clear what I meant.
What I had on my mind was:
- in this PR simply create a new metrics.go file and move the existing initCounter metric to that file
- what you add in this file, move to your original PR, though add it to this newly created metrics.go file

Does that make sense?

Reply:
@wojtek-t  Sorry for misunderstanding it. Updated, PTAL

Reply:
No need to sorry - thanks a lot for updating!